Makefile

```
@@ -1,5 +1,5 @@
 # Cross-platform build system for reviewtask
-.PHONY: build build-all clean test version help
+.PHONY: build build-all clean test version help lint vet ci
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)